babel/numbers.py

```diff
@@ -7,7 +7,7 @@
     The default locale for the functions in this module is determined by the
     following environment variables, in that order:
 
-     * ``LC_NUMERIC``,
+     * ``LC_NUMERIC`` or ``LC_MONETARY`` for currency related functions,
```

Comment:
This documentation doesn't seem to match the new behavior; it's not an "or" if the envvar is set. This should probably say 

```suggestion
     * ``LC_MONETARY`` for currency related functions,
     * ``LC_NUMERIC`` for other numbers,
```
or something along those lines.